runtime/doc/autocmd.txt

```diff
@@ -977,6 +979,15 @@ InsertLeavePre			Just before leaving Insert mode.  Also when
 							*InsertLeave*
 InsertLeave			Just after leaving Insert mode.  Also when
 				using CTRL-O |i_CTRL-O|.  But not for |i_CTRL-C|.
+							*KeyInputPre*
+KeyInputPre			When a key is typed. The pattern is
+				matched against `'current_mode'`.
```

Comment:
Why is `current_mode` quoted?

Reply:
Something like this?
```suggestion
				matched against a string that indicates the
				current mode, which is the same as that is
				returned by `mode()`.
```
(Sorry if my English is not correct.)

Reply:
It may be better to use `mode(1)` here to be more accurate, otherwise it seems OK.

Reply:
Updated.

Reply:
Fixed.